.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
```

Comment:
Label should be done used the github labels right? is not good for the user to have to type the labels (same for assigned)?

Reply:
done